contributing.md

```diff
@@ -18,7 +18,7 @@ Don't forget to give yourself credit! Make sure you add yourself to the contribu
 
 Either:
 * Invoke the [@all-contributors bot](https://allcontributors.org/docs/en/bot/usage) by commenting on your Pull Request or Issue.
-* Shallow clone repo and exeucte `all-contributors add <YOUR_GITHUB_HANDLE> <CONTRIBUTION_TYPE>`
+* Shallow clone repo and execute `all-contributors add <YOUR_GITHUB_HANDLE> <CONTRIBUTION_TYPE>`
```

Comment:
Is this possible? I mean run some `all-contributors` executable? How got that exe on my computer?
I fear this is wrong :grimacing: